apps/nowcasting-app/components/map/pvLatestMap.tsx

```diff
@@ -17,8 +16,11 @@ import { FeatureCollection } from "geojson";
 import { safelyUpdateMapData } from "../helpers/mapUtils";
 import { components } from "../../types/quartz-api";
 import { generateGeoJsonForecastData } from "../helpers/data";
+import dynamic from "next/dynamic";
```

Comment:
whats this doing?

Reply:
Yep this is the NB comment in PR description, it's a weird SSR thing I needed to fix to push the code